assessment_module_manager/assessment_module_manager/module/request_to_module.py

```diff
@@ -32,23 +34,37 @@ async def find_module_by_name(
     return None
 
 
-async def request_to_module(module: Module, path: str, data: dict) -> ModuleResponse:
+async def request_to_module(module: Module, module_config: Optional[str], path: str, data: Optional[dict], method: str) -> ModuleResponse:
     """
     Helper function to send a request to a module.
     It raises appropriate FastAPI HTTPException if the request fails.
     """
     module_secret = env.MODULE_SECRETS[module.name]
+    headers = {'X-API-Secret': module_secret} if module_secret else None
+    if module_config:
+        if headers is None:
+            headers = {}
+        headers['X-Module-Config'] = module_config
```

Comment:
```suggestion
headers = {}
if module_secret:
    headers['X-API-Secret'] = module_secret
if module_config:
    headers['X-Module-Config'] = module_config
```

Does something break if `headers` is `{}` instead of None? If no, the above simplification looks slightly better to me.

Reply:
That is probably fine, I will test it. Looks also better to me